include/class.file.php

```diff
@@ -776,9 +791,13 @@ function read($amount=0, $offset=0) {
     /**
      * Convenience method to send all the file to standard output
      */
-    function passthru() {
```

Comment:
In PHP7 you need to change the original function with same parameters to prevent Warnings.
Line 967 needs to be changed to : passthru($filter=false) also.
Else, thank you for the fix, it works for me.